### ISSUE TITLE
Mounting Boreas/Gulfaxi look like Hallowed/Holmgang in logs

### ISSUE THREAD

**Issue by quisquous** [ravahn/FFXIV_ACT_Plugin#178]
```
[22:47:55.980] 15:1048638C:Tini Poutini:1E:Hallowed Ground:1048638C:Tini Poutini:126:1E0000:1C:48000:0:0:0:0:0:0:0:0:0:0:0:0:64893:64893:4560:4560:1000:1000:201.3474:-230.6911:112.5827:64893:64893:4560:4560:1000:1000:201.3474:-230.6911:112.5827:
[22:47:55.000] 00:082b:You mount Gullfaxi.
[22:47:59.856] 15:1048638C:Tini Poutini:2B:Holmgang:1048638C:Tini Poutini:126:2B0000:1C:48000:0:0:0:0:0:0:0:0:0:0:0:0:64893:64893:4560:4560:1000:1000:201.3474:-230.6911:112.5827:64893:64893:4560:4560:1000:1000:201.3474:-230.6911:112.5827:
[22:47:59.000] 00:082b:You mount Boreas.
```

See: https://github.com/quisquous/cactbot/issues/191

**Comment by panicstevenson** [comment]
Adding to this, it seems this issue plagues more than these specific skills and specific mounts, e.g.:

```code
20|2018-11-16T20:49:08.2990000-06:00|1066038F|Name Naname|AD|Resurrection|1066038F|Name Naname||86ee0488255aca68bcc9c515451266e5
21|2018-11-16T20:49:09.2120000-06:00|1066038F|Name Naname|AD|Resurrection|1066038F|Name Naname|126|AD0000|1C|48000|0|0|0|0|0|0|0|0|0|0|0|0|59790|59790|9480|9480|1000|1000|-108.0399|-40.91711|16.02|59790|59790|9480|9480|1000|1000|-108.0399|-40.91711|16.02||6da70aa227d0395e37f07504fc9a69f8
00|2018-11-16T20:49:09.0000000-06:00|082b||You mount the Model O.|f9a736e1572aea5f4dc099e77342e110
20|2018-11-16T20:51:12.2540000-06:00|1066038F|Name Naname|5E|Elusive Jump|1066038F|Name Naname||d77cab5b1d658d86d5c20f717b1382a6
21|2018-11-16T20:51:13.1670000-06:00|1066038F|Name Naname|5E|Elusive Jump|1066038F|Name Naname|126|5E0000|1C|48000|0|0|0|0|0|0|0|0|0|0|0|0|59790|59790|9480|9480|1000|1000|-108.0399|-40.91711|16.02|59790|59790|9480|9480|1000|1000|-108.0399|-40.91711|16.02||0eecc6cc8d87950bcf548235e6e567d7
00|2018-11-16T20:51:13.0000000-06:00|082b||You mount the Whisper A-go-go.|e2235cb5ebd86dc2f7302516495eec90
```

**Comment by ravahn** [comment]
This is fixed in 2.0+ of the FFXIV Plugin.   The code now distinguishes between skills, mount, and item usage.  It doesn't currently display the mount name or item name, but that will happen in the future.
